driver/config.go

```diff
@@ -28,51 +29,102 @@ type Config struct {
 // NewConfigFromCliContext creates a new config instance from
 // the command line inputs.
 func NewConfigFromCliContext(c *cli.Context) (*Config, error) {
-	jwtSecret, err := jwt.ParseSecretFromFile(c.String(flags.JWTSecret.Name))
-	if err != nil {
-		return nil, fmt.Errorf("invalid JWT secret file: %w", err)
+	// Defaults config
+	cfg := Config{
+		ClientConfig: &rpc.ClientConfig{
+			// TODO: To be confirmed whether taiko addresses in L1 or L2 are constant
+			TaikoL1Address: common.HexToAddress(c.String(flags.TaikoL1Address.Name)),
+			TaikoL2Address: common.HexToAddress(c.String(flags.TaikoL2Address.Name)),
+			Timeout:        12 * time.Second,
+		},
+		RetryInterval:         backoff.DefaultMaxInterval,
+		P2PSyncVerifiedBlocks: false,
+		P2PSyncTimeout:        1 * time.Hour,
+		RPCTimeout:            12 * time.Second,
+		MaxExponent:           0,
 	}
 
-	var (
-		p2pSyncVerifiedBlocks = c.Bool(flags.P2PSyncVerifiedBlocks.Name)
-		l2CheckPoint          = c.String(flags.CheckPointSyncURL.Name)
-	)
+	// Load config file
+	if file := c.String(flags.ConfigFile.Name); file != "" {
+		if err := utils.LoadConfigFile(file, &cfg); err != nil {
+			return nil, fmt.Errorf("%w", err)
+		}
+	}
+	// Apply flag value
+	err := ApplyFlagValue(c, &cfg)
+	if err != nil {
```

Comment:
```suggestion
	if err := ApplyFlagValue(c, &cfg); err != nil {
```

If the function only return an error, lets use `if err := ...; err !=nil {`